frontend/src/globals/unity-util.ts

```diff
@@ -861,6 +874,105 @@ export class UnityUtil {
 		UnityUtil.toUnity('StopClipEdit', undefined, undefined);
 	}
 
+	/**
```

Comment:
Can we add the `@category` tag in these function comments? We run this file through typedoc (https://typedoc.org/guides/overview/) so these comments needs to conform to the schema so it shows up nicely [in our documentation](https://3drepo.github.io/3drepo.io/frontend/classes/unity_util.UnityUtil.html)

Reply:
Hi @carmenfan, done! I went through the file and added comments to methods from other features that didn't have `@category` and were not `@hidden` too.